src/Avalonia.Base/Diagnostics/Diagnostic.Metrics.cs

```diff
@@ -0,0 +1,94 @@
+using System.Diagnostics;
+using System.Diagnostics.Metrics;
+using Avalonia.Interactivity;
+using Avalonia.Threading;
+using Avalonia.Utilities;
+
+namespace Avalonia.Diagnostics;
+
+internal static partial class Diagnostic
+{
+    private static Histogram<double>? s_compositorRender;
+    private static Histogram<double>? s_compositorUpdate;
+    private static Histogram<double>? s_layoutMeasure;
+    private static Histogram<double>? s_layoutArrange;
+    private static Histogram<double>? s_layoutRender;
+    private static Histogram<double>? s_layoutInput;
+
+    public static void InitMetrics()
+    {
+        // Metrics
+        var meter = new Meter("Avalonia.Diagnostic.Meter");
```

Comment:
Shouldn't that be _Avalonia.Diagnostic*s*.Meter_ or _Avalonia*.Diagnostics*.Diagnostic.Meter_?

Reply:
This name is public (for listeners, and also devtools UI), so I made it shorter. I don't have strong opinion though.